node/node.go

```diff
@@ -170,6 +169,7 @@ func MakeFull(log logging.Logger, rootDir string, cfg config.Local, phonebookAdd
 	if node.devMode {
 		cfg.DisableNetworking = true
 	}
+	node.config = cfg
```

Comment:
```suggestion
		node.config.DisableNetworking = true
	}
```

Reply:
I was hoping to avoid setting this field twice - instead, I think that we should try and have a single copy of the `config.Local`. Until we do this, I think that ensuring that we "just" copy it here would make the next change *slightly* easier.

Reply:
Oops, I didn't realize cfg was used below as well.